doc/en/announce/release-5.2.4.rst

```diff
@@ -0,0 +1,22 @@
+pytest-5.2.4
+=======================================
+
+pytest 5.2.4 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/latest/changelog.html.
+
+Thanks to all who contributed to this release, among them:
+
+* Anthony Sottile
+* Bruno Oliveira
+* Daniel Hahler
+* Hugo
+* Michael Shields
```

Comment:
Seems like this list is off?

Reply:
yeah no idea how towncrier computes this -- perhaps it includes reporters of associated issues?

Reply:
It is actually computed by `release.py`:

https://github.com/pytest-dev/pytest/blob/19a15a94eeb403631f847fab5fe5073bc56e8bcc/scripts/release.py#L16-L26

@asottile please make sure you have the tags locally.

Reply:
I do have them -- here's what it reports:

```console
$ git describe --abbrev=0 --tags
5.2.3
$ git log 5.2.3..HEAD --format=%aN
Anthony Sottile
Anthony Sottile
Anthony Sottile
Anthony Sottile
Bruno Oliveira
Daniel Hahler
Bruno Oliveira
Michael Shields
Hugo
Hugo
```

which is weird given:

```console
$ git log --graph --oneline --decorate
* c9a96cdee (HEAD -> release-5.2.4, asottile/release-5.2.4) Preparing release version 5.2.4
*   19a15a94e (origin/master, origin/HEAD, master) Merge pull request #6197 from asottile/fix_init_py_discovery
|\  
| * 4e0f99260 (asottile/fix_init_py_discovery, fix_init_py_discovery) Add regression tests for __init__.py breakage
| * 176c7771f Revert "fix bug with nonskipped first test in package (#5831)"
|/  
*   e856638ba Preparing release version 5.2.3 (#6190)
|\  
| * dd9a27cf5 (tag: 5.2.3) Adjust CHANGELOG
```

Reply:
The new tag is 5.2.4.

Reply:
yes, this command is "what has changed since the last tag"

Reply:
Ah.. that's because the tag is not on the master branch I think.